constraint: true
solve_time: 0.5
turning_radius: 1
max_dist: 4
# grow_size: 800
limit_time: true
obstacle_resolution: 0.001
base_frame: robot0